slides/testing/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Yes. We will have to create two versions or a some kind of flag I guess. For development it currently runs in watch mode.